### PR TITLE
8028265: Add legacy tz tests to OpenJDK

### PR DESCRIPTION
**Pull request by satoyoshiki**
Please review this PR.  The PR open sources the closed timezone tests.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8028265](https://bugs.openjdk.org/browse/JDK-8028265): Add legacy tz tests to OpenJDK

### Reviewers
 * [Sean Coffey](https://openjdk.org/census#coffeys) (@coffeys - **Reviewer**)
 * [Naoto Sato](https://openjdk.org/census#naoto) (@naotoj - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/9476/head:pull/9476` \
`$ git checkout pull/9476`

Update a local copy of the PR: \
`$ git checkout pull/9476` \
`$ git pull https://git.openjdk.org/jdk pull/9476/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9476`

View PR using the GUI difftool: \
`$ git pr show -t 9476`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/9476.diff">https://git.openjdk.org/jdk/pull/9476.diff</a>

</details>

### CODE REVIEW

**Comment by satoyoshiki** [comment]
Can I ask you to help reviewing this PR?
@coffeys @naotoj

File: test/jdk/java/util/TimeZone/tools/share/Makefile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#
+#
+# This makefile is used to update the time zone tests data: aliases.txt displaynames.txt
+# and VERSION. After switching to new build system in JDK8 the following make
+# command should be used for test data updates:
+#     make JDK_HOME=<path to built JDK with latest tzdata>
+#     make install
+#
+
+TZDATA = ../../../../../../../../open/src/java.base/share/data/tzdata
```
Comment (coffeys) [inline test/jdk/java/util/TimeZone/tools/share/Makefile:33]: suggest you replace this with `../../../../../../../src/java.base/share/data/tzdata/`

**Comment by coffeys** [comment]
1 minor comment. Looks good to me.

File: test/jdk/java/util/TimeZone/tools/share/makeZoneData.pl [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+
+#
+#
+#
+# Perl script to generate ZoneData from Olson public time zone data.
+#
+# For J2SE before JDK1.3, see ../../README how to update TimeZone.java 
+# static TimeZoneData.
+# For J2SE since JDK1.4, this script is used to generate testdata(reference)
+# for ZoneData.sh which is one of TimeZone Regression test.
```
Comment (coffeys) [inline test/jdk/java/util/TimeZone/tools/share/makeZoneData.pl:32]: suggest that you remove these lines - redundant information.

**Pull request closed without merging**